.github/workflows/default_branch_verification.yml

```diff
@@ -20,5 +20,7 @@ jobs:
 
       - name: Grant execute permission for gradlew
         run: chmod +x gradlew
+      - name: Analyze
+        run: ./gradlew ktlintCheck
```

Comment:
Could you look at the logs from action for example for this PR 
![image](https://user-images.githubusercontent.com/57491794/218968854-3ea0a551-93e5-47bc-953c-42c82f1c81fb.png)
Does this indicate we have already `ktlint` in the `build`? If we will don't use the `build` for GitHub action and w use `assemble` then we should consider writing jobs for `ktlint`, `lint`, `tests`

Reply:
This logs was added when you create PR #32

Reply:
These logs are about building our `ktlint-ruleset` module where we will put custom rules